app/javascript/entrypoints/application.ts

```diff
@@ -1,9 +1,8 @@
-import { start } from 'mastodon/common';
 import { loadLocale } from 'mastodon/locales';
 import main from 'mastodon/main';
 import { loadPolyfills } from 'mastodon/polyfills';
 
-start();
```

Comment:
If `common` is included in the app layout, do we really need to call it from every other entrypoint?